__tests__/RacingGame.test.js

```diff
@@ -1,27 +1,31 @@
-import Car from '../src/Car';
-import RacingGame from '../src/RacingGame';
+import Car from '../src/domain/Car';
+import CarList from '../src/domain/CarList';
+import RacingGame from '../src/domain/RacingGame';
 
 describe('RacingGame 테스트', () => {
   const carNames = ['자동차1', '자동차2', '자동차3'];
+  const checkCanMoveForward = () => true;
   let cars = null;
   let racingGame = null;
 
   beforeEach(() => {
     cars = carNames.map((name) => new Car(name));
-    racingGame = new RacingGame(cars);
+    racingGame = new RacingGame(carNames);
   });
 
   describe('레이싱 게임 세팅 테스트', () => {
     test('경주에 참여할 자동차들이 있다.', () => {
-      expect(racingGame.cars.every((car) => car instanceof Car)).toBe(true);
+      expect(racingGame.carList instanceof CarList).toBe(true);
     });
   });
 
   describe('자동차 게임 전진 테스트', () => {
     test('자동차 전진 조건을 만족하면 자동차는 전진한다.', () => {
-      racingGame.startRace({ checkCanMoveForward: () => true });
+      racingGame.startRace(checkCanMoveForward);
 
-      expect(racingGame.cars.every((car) => car.distanceDriven > 0)).toBe(true);
+      expect(
+        racingGame.carList.cars.every((car) => car.distanceDriven > 0)
+      ).toBe(true);
```

Comment:
```suggestion
      expect(
        racingGame.carList.cars.every((car) => car.distanceDriven === 1)
      ).toBe(true);
```
아예 이렇게 상태 자체를 비교하는 것도 좋을 것 같아요!
결과를 더 정확하게 예측하는거죠 ㅎㅎ